packages/web/src/components/notification/Notification/utils.ts

```diff
@@ -28,10 +29,8 @@ export const getRankSuffix = (rank: number) => {
   return 'th'
 }
 
-export const getTwitterHandleByUserHandle = async (userHandle: string) => {
-  const { twitterHandle } = await audiusBackendInstance.getSocialHandles(
-    userHandle
-  )
+export const getTwitterHandleByUserHandle = async (user: User) => {
+  const { twitterHandle } = await audiusBackendInstance.getSocialHandles(user)
```

Comment:
If I understand correctly, `getSocialHandles` now returns an empty object if the `user` has those values on it already. Would that not break this utility and the code relying on it if the cached user has already had social handles fetched?

Reply:
changed to return the passed in user if it already has social handles. this code will be all thrown away soon anyway. early next week